pkg/kubectl/cmd/rollout/rollout_pause.go

```diff
@@ -125,7 +119,7 @@ func (o *PauseConfig) CompletePause(f *cmdutil.Factory, cmd *cobra.Command, out
 	return nil
 }
 
-func (o PauseConfig) RunPause() error {
+func (o PauseConfig) RunPause() {
```

Comment:
RunPause needs to continue return an error, you just need to checkErr in the caller site
https://github.com/kubernetes/kubernetes/blob/master/docs/devel/kubectl-conventions.md#command-implementation-conventions